dataduct/utils/hook.py

```diff
@@ -0,0 +1,50 @@
+"""Hook helpers
+"""
+import os
+import imp
+
+
+def default_before_hook(*args, **kwargs):
+    """The default before hook, will act like it's not even there
+    """
+    return args, kwargs
+
+
+def default_after_hook(result):
```

Comment:
Result will never be unfilled, as
```
result = func(*new_args, **new_kwargs)
new_result = after_hook(result)
```
Is the only place its used.

Better to make sure it's always filled imo.